### PR TITLE
Improvement/update modelizer page management

### PR DESCRIPTION
**Pull request by k-y0u**
* Create new route for ModelsPage
* Update Modelizer routes
* Delete ViewSwitchEvent
* Rename files :
  * ModelizerModelsView to ModelsView
  * ModelizerNavigationBar to NavigationBar
  * ModelizerModelView to ModelizerDrawView
* Update e2e tests

### CODE REVIEW

File: src/pages/ModelizerPage.vue [hunk 1]
```diff
@@ -13,11 +13,11 @@
         data-cy="modelizer-page"
       >
         <modelizer-model-view
-          v-show="viewType === 'draw'"
+          v-if="viewType === 'draw'"
           :project-name="projectName"
         />
         <modelizer-text-view
-          v-show="viewType === 'text'"
+          v-if="viewType === 'text'"
```
Comment (Zorin95670) [inline src/pages/ModelizerPage.vue:20]: v-else-if

Comment (k-y0u) [reply src/pages/ModelizerPage.vue]: done

**Review by justinehell — CHANGES_REQUESTED** [review]

File: src/components/ModelizerTextView.vue [hunk 2]
```diff
@@ -243,49 +210,24 @@ async function getModel() {
...
-/**
- * On switch to 'text' view, update project files and send CreateFileNode event for each
- * files created after calling renderPluginFiles.
- * Also send UpdateFileContent event if renderPluginFiles returns any updated files.
- * @param {String} newViewType - Updated view type.
- * @return {Promise<void>} Promise with nothing on success otherwise an error.
- */
-async function onSwitchView(viewType) {
-  if (viewType === 'draw') {
+async function onSelectFileTab(event) {
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply src/components/ModelizerTextView.vue:215]: done

File: src/pages/ModelsPage.vue [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import DeleteModelDialog from 'components/dialog/DeleteModelDialog';
+import RenameModelDialog from 'components/dialog/RenameModelDialog';
+import ImportModelTemplateDialog from 'components/dialog/ImportModelTemplateDialog';
+
+const route = useRoute();
+const projectName = computed(() => route.params.projectName);
+</script>
+
+<style scoped>
+.q-layout.modelizer-page>.q-page-container {
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply src/pages/ModelsPage.vue]: done

File: cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature [hunk 4]
```diff
@@ -10,15 +9,24 @@ Feature: Test modelizer model view: plugin initialization
...
+    When I set on '[data-cy="create-model-form"] [data-cy="name-input"]' text 'modelName'
+    And  I click on '[data-cy="create-model-form"] [data-cy="submit-button"]'
+    Then I expect current url is 'projectName/modelizer/draw\?path=terrator-plugin/modelName'
 
-  Scenario Outline: Set text as '<filter>'should display only one element
     When I click on '[data-cy="component-defnitions-item_terrator-plugin"]'
+    And  I wait 1 second
     Then I expect '[class*="plugin-definitions"]' appear 1 time on screen
-    And  I expect '[class*="component-definition-card"]' appear 18 times on screen
+    Then I expect '[data-cy="component-defnitions-item_terrator-plugin"] [class*="component-definition-card"]' appear 18 times on screen
```
Comment (Zorin95670) [inline cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature:25]: And

File: cypress/e2e/ModelizerPage/SwitchView/delete.feature [hunk 5]
```diff
@@ -1,4 +1,3 @@
-@skip
 # TODO: update/fix test
```
Comment (justinehell) [inline cypress/e2e/ModelizerPage/SwitchView/delete.feature:1]: remove if fixed

File: cypress/e2e/ModelizerPage/TextView/file/add.file.feature [hunk 6]
```diff
@@ -13,10 +13,18 @@ Feature: Test modelizer text view: add file
...
+    When I click on '[data-cy="create-model-button"]'
+    Then I expect '[data-cy="create-model-form"] [data-cy="plugin-select"]' is 'terrator-plugin'
+
+    When I set on '[data-cy="create-model-form"] [data-cy="name-input"]' text 'modelName'
+    And  I click on '[data-cy="create-model-form"] [data-cy="submit-button"]'
+    Then I expect current url is '{{ projectName }}/modelizer/draw\?path=terrator-plugin/modelName'
+
+    When I click on '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="false"]'
     Then I expect '[data-cy="file-explorer"] [data-cy="folder_{{ projectName }}"]' exists
     And  I wait 2 seconds
```
Comment (justinehell) [inline cypress/e2e/ModelizerPage/TextView/file/add.file.feature:29]: 2 seconds

File: cypress/e2e/ModelizerPage/TextView/file/create.file.feature [hunk 7]
```diff
@@ -13,10 +13,18 @@ Feature: Test modelizer text view: create file and folder
...
+    When I click on '[data-cy="create-model-button"]'
+    Then I expect '[data-cy="create-model-form"] [data-cy="plugin-select"]' is 'terrator-plugin'
+
+    When I set on '[data-cy="create-model-form"] [data-cy="name-input"]' text 'modelName'
+    And  I click on '[data-cy="create-model-form"] [data-cy="submit-button"]'
+    Then I expect current url is '{{ projectName }}/modelizer/draw\?path=terrator-plugin/modelName'
+
+    When I click on '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="false"]'
     Then I expect '[data-cy="file-explorer"] [data-cy="folder_{{ projectName }}"]' exists
     And  I wait 2 seconds
```
Comment (justinehell) [inline cypress/e2e/ModelizerPage/TextView/file/create.file.feature:29]: 2 seconds

File: cypress/e2e/ModelizerPage/TextView/file/open.file.feature [hunk 8]
```diff
@@ -14,13 +14,25 @@ Feature: Test modelizer text view: open file
...
+
+    When I set on '[data-cy="create-model-form"] [data-cy="name-input"]' text 'modelName'
+    And  I click on '[data-cy="create-model-form"] [data-cy="submit-button"]'
+    Then I expect current url is '{{ projectName }}/modelizer/draw\?path=terrator-plugin/modelName'
+
+    # Go to text view and check files
+    When I click on '[data-cy="modelizer-switch-button"] [aria-pressed="false"]'
+    Then I expect current url is '{{projectName}}/modelizer/text\?path=terrator-plugin/{{modelName}}'
+    And  I expect '[data-cy="file-explorer"] [data-cy="folder_{{projectName}}"]' is '{{projectName}}'
     And  I wait 2 seconds
```
Comment (justinehell) [inline cypress/e2e/ModelizerPage/TextView/file/open.file.feature:32]: 2 seconds

File: cypress/e2e/ModelizerPage/TextView/file/open.file.feature [hunk 9]
```diff
@@ -14,13 +14,25 @@ Feature: Test modelizer text view: open file
...
-    And  I click on '[data-cy="file-explorer"] [data-cy="folder_{{ projectName }}"]'
+
+    When   I click on '[data-cy="file-explorer"] [data-cy="folder_{{ projectName }}"]'
+    Then   I expect '[data-cy="file-explorer"] [data-cy="file_README.md"]' exists
```
Comment (justinehell) [inline cypress/e2e/ModelizerPage/TextView/file/open.file.feature:35]: remove 1 space after `When` and |Then`

File: cypress/e2e/RoundTrips/create.project.template.files.feature [hunk 10]
```diff
@@ -33,16 +37,20 @@ Feature: Test roundtrip of the application : create project with template files
...
+    Then I expect '[data-cy="file-explorer"] [data-cy="file_terrator-plugin/infra2/main.tf"]' exists
 
     When I double click on '[data-cy="file-explorer"] [data-cy="file_terrator-plugin/infra2/main.tf"]'
     Then I expect '[data-cy="active-tab"] [data-cy="file_terrator-plugin/infra2/main.tf"]' exists
     And  I expect active file content to be equal to 'cypress/resources/infra2/main.tf'
 
     When I click on '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin/infra1"]'
-    And  I double click on '[data-cy="file-explorer"] [data-cy="file_terrator-plugin/infra1/main.tf"]'
+    Then I expect '[data-cy="file-explorer"] [data-cy="file_terrator-plugin/infra1/main.tf"]' exists
+    Then I expect '[data-cy="file-explorer"] [data-cy="file_terrator-plugin/infra1/provider.tf"]' exists
```
Comment (justinehell) [inline cypress/e2e/RoundTrips/create.project.template.files.feature:51]: Then -> And

File: cypress/e2e/RoundTrips/delete.project.feature [hunk 11]
```diff
@@ -17,12 +18,21 @@ Feature: Test homepage: project deletion
...
+    And  I click on '[data-cy="create-model-form"] [data-cy="submit-button"]'
+    Then I expect current url is '{{projectName}}/modelizer/draw\?path=terrator-plugin/{{modelName}}'
 
     # Check project root folder is created in Text view
-    When I visit the '/#/modelizer/{{projectName}}/text'
-    Then I expect '[data-cy="file-explorer"] [data-cy="folder_{{ projectName }}"]' exists
+    When I click on '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="false"]'
     And  I wait 1 second
+    Then I expect current url is '{{projectName}}/modelizer/text\?path=terrator-plugin/{{modelName}}'
+    Then I expect '[data-cy="file-explorer"] [data-cy="folder_{{ projectName }}"]' exists
```
Comment (justinehell) [inline cypress/e2e/RoundTrips/delete.project.feature:35]: Then -> And

File: cypress/e2e/RoundTrips/rename.project.feature [hunk 12]
```diff
@@ -20,19 +20,19 @@ Feature: Test roundtrip of the application : rename project
...
-    Then I expect current url is 'modelizer/projectTest/text\?path=terrator-plugin/{{modelName}}'
+    And  I wait 1 second
+    Then I expect current url is 'projectTest/modelizer/text\?path=terrator-plugin/{{modelName}}'
     And  I expect '[data-cy="file-explorer"] [data-cy="folder_projectTest"]' is 'projectTest'
 
     When I click on '[data-cy="file-explorer"] [data-cy="folder_projectTest"]'
     And  I expect '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin"]' exists
 
     When I click on '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin"]'
     And  I expect '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin/{{modelName}}"]' exists
```
Comment (justinehell) [inline cypress/e2e/RoundTrips/rename.project.feature:35]: And -> Then

File: cypress/e2e/roundtrip.feature [hunk 13]
```diff
@@ -1,6 +1,6 @@
 Feature: Test roundtrip of the application
 
-  @skip
+  # @skip
   # TODO: update/fix test
```
Comment (justinehell) [inline cypress/e2e/roundtrip.feature:4]: remove if fixed

**Review by justinehell — CHANGES_REQUESTED** [review]

File: cypress/e2e/ModelizerPage/Common/switch.view.feature [hunk 14]
```diff
@@ -8,36 +8,37 @@ Feature: Test modelizer page: switch view (text/model)
...
-    When I visit the '/'
-    And  I visit the '/#/modelizer/projectName'
-    Then I expect current url is '/modelizer/projectName/model'
+    # Model creation
+    When I click on '[data-cy="create-model-button"]'
+    Then I expect '[data-cy="create-model-form"] [data-cy="plugin-select"]' is 'terrator-plugin'
+
+    When I set on '[data-cy="create-model-form"] [data-cy="name-input"]' text 'modelName'
+    And  I click on '[data-cy="create-model-form"] [data-cy="submit-button"]'
+    Then I expect current url is 'projectName/modelizer/draw\?path=terrator-plugin/modelName'
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/Common/switch.view.feature:21]: Done

File: cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature (hunk 4 above)
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature]: Done

File: src/components/ModelizerNavigationBar.vue [hunk 15]
```diff
@@ -150,10 +153,15 @@ async function upload() {
  *
  * @param {string} newViewType
  */
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply src/components/ModelizerNavigationBar.vue]: done

File: cypress/e2e/ModelizerPage/Common/switch.view.feature [hunk 16]
```diff
@@ -8,36 +8,37 @@ Feature: Test modelizer page: switch view (text/model)
...
 
-  @skip
   # TODO: update/fix test
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/Common/switch.view.feature]: done

File: cypress/e2e/ModelizerPage/Common/switch.view.feature [hunk 17]
```diff
@@ -8,36 +8,37 @@ Feature: Test modelizer page: switch view (text/model)
...
   Scenario: Modelizer 'text' page should load the correct content
-    When I visit the '/#/modelizer/projectName/text'
+    When I click on '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="false"]'
     Then I expect '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="true"] [class="block"]' is 'Text'
     And  I expect '[data-cy="modelizer-text-view"]' exists
     And  I expect '[data-cy="file-explorer"]' exists
     And  I expect '[data-cy="file-tabs"]' exists
 
-  @skip
   # TODO: update/fix test
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/Common/switch.view.feature]: done

File: cypress/e2e/ModelizerPage/ModelView/plugin/display.component.feature [hunk 18]
```diff
@@ -1,4 +1,3 @@
-@skip
 # TODO: update/fix test
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/ModelView/plugin/display.component.feature]: done

File: cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature [hunk 19]
```diff
@@ -1,4 +1,3 @@
-@skip
 # TODO: update/fix test
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature]: done

File: cypress/e2e/ModelizerPage/ModelView/plugin/init.plugin.feature [hunk 20]
```diff
@@ -1,4 +1,3 @@
-@skip
 # TODO: update/fix test
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/ModelView/plugin/init.plugin.feature]: done

File: cypress/e2e/ModelizerPage/SwitchView/add.component.feature [hunk 21]
```diff
@@ -1,4 +1,3 @@
-@skip
 # TODO: update/fix test
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/SwitchView/add.component.feature]: done

File: cypress/e2e/roundtrip.feature (hunk 13 above)
Comment (k-y0u) [reply cypress/e2e/roundtrip.feature]: done

File: cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature [hunk 22]
```diff
@@ -40,12 +48,8 @@ Feature: Test modelizer model view: plugin initialization
       | aws_key_pair          |
 
   Scenario Outline: Set text as '<filter>' should display only two elements
-    When I click on '[data-cy="component-defnitions-item_terrator-plugin"]'
-    Then I expect '[class*="plugin-definitions"]' appear 1 time on screen
-    And  I expect '[class*="component-definition-card"]' appear 18 times on screen
-
     When I set on '[data-cy="definitions-filter-input"]' text '<filter>'
-    And  I expect '[class*="component-definition-card"]' appear 2 times on screen
+    Then I expect '[data-cy="component-defnitions-item_terrator-plugin"] [class*="component-definition-card"]' appear 2 time on screen
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature]: done

File: cypress/e2e/ModelizerPage/SwitchView/add.component.feature [hunk 23]
```diff
@@ -10,71 +9,94 @@ Feature: Test switch model to text view: add component/link
...
-    And  I expect '[data-cy="file-tabs-container"] [data-cy="active-tab"]' is 'leto-modelizer.config.json'
-    And  I expect active file content to contain '{.*"terrator-plugin":.*{}}'
 
-    When I click on '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="false"]'
-    Then I expect '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="true"] [class="block"]' is 'Model'
-    And  I expect '[data-cy="modelizer-model-view"] [data-cy="draw-container"]' exists
+    When I click on '[data-cy="file-explorer"] [data-cy="folder_projectName"]'
+    And  I click on '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin"]'
+    And  I click on '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin/modelName"]'
+    Then I expect '[data-cy="file_terrator-plugin/modelName/new_file.tf"]' appear 1 times on screen
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/SwitchView/add.component.feature]: done

File: cypress/e2e/ModelizerPage/SwitchView/add.component.feature [hunk 24]
```diff
@@ -10,71 +9,94 @@ Feature: Test switch model to text view: add component/link
...
+    And  I wait 1 second
     Then I expect '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="true"] [class="block"]' is 'Text'
     And  I expect '[data-cy="modelizer-text-view"]' exists
-    And  I expect '[data-cy="file_new_file.tf"]' appear 2 times on screen
-    And  I expect '[data-cy="file_leto-modelizer.config.json"]' appear 2 times on screen
+
+    When I click on '[data-cy="file-explorer"] [data-cy="folder_projectName"]'
+    And  I click on '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin"]'
+    And  I click on '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin/modelName"]'
+    Then I expect '[data-cy="file_terrator-plugin/modelName/new_file.tf"]' appear 1 times on screen
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/SwitchView/add.component.feature]: done

File: cypress/e2e/ModelizerPage/SwitchView/add.component.feature [hunk 25]
```diff
@@ -10,71 +9,94 @@ Feature: Test switch model to text view: add component/link
...
+    And  I expect '[data-cy="file_terrator-plugin/modelName/leto-modelizer.config.json"]' appear 1 times on screen
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/SwitchView/add.component.feature]: done

File: cypress/e2e/ModelizerPage/SwitchView/add.component.feature [hunk 26]
```diff
@@ -85,30 +107,54 @@ Feature: Test switch model to text view: add component/link
...
     When I click on '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="false"]'
+    And  I wait 1 second
     Then I expect '[data-cy="navigation-bar"] [data-cy="modelizer-switch-button"] [aria-pressed="true"] [class="block"]' is 'Text'
     And  I expect '[data-cy="modelizer-text-view"]' exists
-    And  I expect '[data-cy="file-tabs-container"] [data-cy="inactive-tab"]' is 'new_file.tf'
+
+    When I click on '[data-cy="file-explorer"] [data-cy="folder_projectName"]'
+    And  I click on '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin"]'
+    And  I click on '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin/modelName"]'
+    Then I expect '[data-cy="file_terrator-plugin/modelName/new_file.tf"]' appear 1 times on screen
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/SwitchView/add.component.feature]: done

File: cypress/e2e/ModelizerPage/SwitchView/add.component.feature [hunk 27]
```diff
@@ -85,30 +107,54 @@ Feature: Test switch model to text view: add component/link
...
+    And  I expect '[data-cy="file_terrator-plugin/modelName/leto-modelizer.config.json"]' appear 1 times on screen
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/SwitchView/add.component.feature]: done

File: cypress/e2e/ModelizerPage/SwitchView/add.component.feature [hunk 28]
```diff
@@ -85,30 +107,54 @@ Feature: Test switch model to text view: add component/link
...
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/SwitchView/add.component.feature]: done

File: cypress/e2e/ModelizerPage/SwitchView/delete.feature (hunk 5 above)
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/SwitchView/delete.feature]: done

File: cypress/e2e/ModelsPage/delete.model.feature [hunk 29]
```diff
@@ -37,15 +37,16 @@ Feature: Test models page: model deletion
 
     When I set on '[data-cy="create-model-form"] [data-cy="name-input"]' text '{{modelName2}}'
     And  I click on '[data-cy="create-model-form"] [data-cy="submit-button"]'
-    Then I expect current url is 'modelizer/{{projectName}}/model\?path=terrator-plugin/{{modelName2}}'
+    Then I expect current url is '{{projectName}}/modelizer/draw\?path=terrator-plugin/{{modelName2}}'
 
     # Go to text view and check files of first model doesn't exists
     When I click on '[data-cy="modelizer-switch-button"] [aria-pressed="false"]'
-    Then I expect current url is 'modelizer/{{projectName}}/text\?path=terrator-plugin/{{modelName2}}'
+    And  I wait 1 seconds
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelsPage/delete.model.feature]: done

File: cypress/e2e/ModelsPage/create.model.feature [hunk 30]
```diff
@@ -10,7 +10,7 @@ Feature: Test models page: model creation
     When I click on '[data-cy="create-project-button"]'
     And  I set on '[data-cy="create-project-form"] [data-cy="name-input"]' text '{{projectName}}'
     And  I click on '[data-cy="create-project-form"] [data-cy="submit-button"]'
-    Then I expect current url is '/#/modelizer/{{projectName}}/models'
+    Then I expect current url is '{{projectName}}/models'
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelsPage/create.model.feature:13]: When I had to update the url, I did it.
Good or bad, when can discuss it ?
I can make a global change in a new commit.
I can also reverse this change on my actual commit.
@Zorin95670 What do you think about it ?

File: cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature [hunk 31]
```diff
@@ -10,15 +9,24 @@ Feature: Test modelizer model view: plugin initialization
...
+    Then I expect current url is 'projectName/modelizer/draw\?path=terrator-plugin/modelName'
 
-  Scenario Outline: Set text as '<filter>'should display only one element
     When I click on '[data-cy="component-defnitions-item_terrator-plugin"]'
+    And  I wait 1 second
     Then I expect '[class*="plugin-definitions"]' appear 1 time on screen
-    And  I expect '[class*="component-definition-card"]' appear 18 times on screen
+    Then I expect '[data-cy="component-defnitions-item_terrator-plugin"] [class*="component-definition-card"]' appear 18 times on screen
 
+  Scenario Outline: Set text as '<filter>'should display only one element
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelizerPage/ModelView/plugin/filter.plugin.feature]: done

File: cypress/e2e/ModelsPage/create.template.feature [hunk 32]
```diff
@@ -10,34 +10,35 @@ Feature: Test models page: model creation from template
...
 
     When I click on '[data-cy="file-explorer"] [data-cy="folder_{{projectName}}"]'
     And  I expect '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin"]' exists
 
     When I click on '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin"]'
-    And  I expect '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin/{{modelName}}"]' exists
-    And  I expect '[data-cy="file-explorer"] [data-cy="file_terrator-plugin/{{modelName}}/leto-modelizer.config.json"]' exists
+    Then I expect '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin/{{modelName}}"]' exists
+    When I click on '[data-cy="file-explorer"] [data-cy="folder_terrator-plugin/{{modelName}}"]'
     And  I expect '[data-cy="file-explorer"] [data-cy="file_terrator-plugin/{{modelName}}/main.tf"]' exists
```
[the remark this answers is not in the document]
Comment (k-y0u) [reply cypress/e2e/ModelsPage/create.template.feature]: done

File: cypress/e2e/RoundTrips/create.project.template.files.feature (hunk 10 above)
Comment (k-y0u) [reply cypress/e2e/RoundTrips/create.project.template.files.feature]: done

File: cypress/e2e/RoundTrips/delete.project.feature (hunk 11 above)
Comment (k-y0u) [reply cypress/e2e/RoundTrips/delete.project.feature]: done

File: cypress/e2e/RoundTrips/rename.project.feature (hunk 12 above)
Comment (k-y0u) [reply cypress/e2e/RoundTrips/rename.project.feature]: done

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**